include/alpaka/api/cuda/executor.hpp

```diff
@@ -44,6 +44,14 @@ namespace alpaka::onAcc::trait
             return internal::cudaHipAtomic;
         }
     };
+    template<>
+    struct GetIntrinsicImpl::Op<alpaka::exec::GpuCuda>
+    {
+        constexpr decltype(auto) operator()(alpaka::exec::GpuCuda const) const
```

Comment:
could you change `constexpr decltype(auto)` to `constexpr int` that the user knows what is the result. It would than follow `std::popcount()`